test/quantization/core/test_workflow_ops.py

```diff
@@ -557,7 +557,7 @@ def test_fixed_qparams_fq_module(self, device, X):
     def test_fq_serializable_per_tensor(self):
         observer = default_observer
         quant_min = 0
-        quant_max = 255
+        quant_max = 127
```

Comment:
wondering why do we change this?

Reply:
the range of default observer is [0, 127] https://github.com/pytorch/pytorch/blob/fbdce8e6caf2de87874e0bc37c12811325849e11/torch/ao/quantization/observer.py#L1415
there's a test error if set quant_max = 255

Reply:
I see, looks like we have quant_min/quant_max for both FakeQuantize module and Observer module, wonder whether we should dedup them or not